src/IECoreRI/RendererImplementation.cpp

```diff
@@ -278,6 +278,18 @@ void IECoreRI::RendererImplementation::setOption( const std::string &name, IECor
 		return;
 	}
 
+
+	if( name == "render:sampleMotion" )
+	{
+		if( m_options->readable().find( "ri:hider:sampleMotion" ) == m_options->readable().end() )
+		{
+			// If we aren't specifying the specific RenderMan sample motion option, then drive it
```

Comment:
Having the order of operations be important to the semantics seems potentially confusing and fragile. Maybe instead we could either say "sampleMotion" is just a straightforward alias to "ri:hider:samplemotion", or store both and resolve them (with the "ri:" one winning as you have here) in `worldEnd()`. I think I prefer the latter...